src/runtime-rs/config/configuration-remote.toml.in

```diff
@@ -0,0 +1,297 @@
+# Copyright 2024 Kata Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+
+# XXX: WARNING: this file is auto-generated.
+# XXX:
+# XXX: Source file: "@CONFIG_REMOTE_IN@"
+# XXX: Project:
+# XXX:   Name: @PROJECT_NAME@
+# XXX:   Type: @PROJECT_TYPE@
+
+
+[hypervisor.remote]
+# Default VM information query service unix domain socket, created by cloud-api-adaptor
+# Ref: https://github.com/confidential-containers/cloud-api-adaptor/blob/main/src/cloud-api-adaptor/docs/vminfo.md
+remote_hypervisor_socket = "/run/peerpod/hypervisor.sock"
+# Timeout in seconds for creating a remote hypervisor, 600s(10min) by default
+remote_hypervisor_timeout = 600
+
+
+# Enable confidential guest support.
+# Toggling that setting may trigger different hardware features, ranging
+# from memory encryption to both memory and CPU-state encryption and integrity.
+# The Kata Containers runtime dynamically detects the available feature set and
+# aims at enabling the largest possible one, returning an error if none is
+# available, or none is supported by the hypervisor.
+#
+# Known limitations:
+# * Does not work by design:
+#   - CPU Hotplug 
+#   - Memory Hotplug
+#   - NVDIMM devices
```

Comment:
It's good to have it documented here that the remote hypervisor doesn't support NVDIMM but it should also be enforced in `validate()`, right?

Reply:
`cloud-hypervisor` and `qemu` also documented that they doesn't support NVDIMM devices, and I cannot find related check in `validate()`. Could you please give me some more details about which one I should check?

Reply:
Hm, qemu supports nvdimm so it's no surprise you can't see a guard against it.  I don't know about clh, if it doesn't support nvdimm and doesn't check then it probably should.